botocore/credentials.py

```diff
@@ -312,6 +312,7 @@ class Credentials:
     :param str token: The security token, valid only for session credentials.
     :param str method: A string which identifies where the credentials
         were found.
+    :param str account_id: The account ID associated with the credentials.
```

Comment:
Forgot to add param comment for `account_id` in previous PR #3328.

Reply:
We should probably briefly note the that this is an optional field. Either in the string or with something like:
```suggestion
    :param str account_id: (optional) An account ID associated with the credentials.
```